features/step_definitions/route.rb

```diff
@@ -131,3 +131,13 @@
       | #{label_filter} |
     })
 end
+
+Given /^dns basedomain is stored in the#{OPT_SYM} clipboard$/ do | cb_name |
```

Comment:
We already have a step `I store default router subdomain in the#{OPT_SYM} clipboard` and it does NOT require admin access. Better use it. It requires you to have a project though. Shouldn't be a big deal comapred to requirement to have admin access.

Reply:
I'd like to get the basedomain here, but actually the subdomain is `"apps." + basedomain` so it is OK for me to use it. Thanks.

Reply:
In such case it is good to create your step , but lets base it on the existing step.